geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -287,6 +287,9 @@ public static String processIncomingClassName(String name) {
     if (name.startsWith(oldPackage)) {
       return newPackage + name.substring(oldPackage.length());
     }
+    if (name.equals("org.apache.geode.cache.query.internal.cq.ServerCQImpl")) {
```

Comment:
This string and the one below it are used twice in this class.  Maybe extract them into `static final` fields?

Reply:
Sure could. Was staying consistent with the other swizzle happening here.